[#147125641] Add status page info to support page

jollery opened the pull request:
## what

Added link to status page in first paragraph and a 'how to keep up to date' section at the bottom. Review comments from Jonathan Glassman are incorporated into the 2nd commit - it's this one that needs to be approved and merged. 

## who can approve

Anyone except me.

jollery committed:
Add status page info to support page

Added link to status page and how users can sign up for alerts

jonathanglassman commented on source/support.html.erb:
```diff
@@ -90,18 +95,17 @@ title: Support
-        for critical incidents.
+        To contact us about a critical issue outside of working hours, please
+        use the emergency email address you have been given. This will raise a
+        high priority alert and could wake us up in the middle of the night, so
```
Instead of saying "...and could wake us up in the middle of the night", maybe say "This is monitored 24 hours a day and will raise a high priority alert, so it should only be used for critical incidents." Just to de-personalise it slightly, unless that's the preferred style.

jollery replied:
The thing we're trying to highlight here is the impact using this email address will have on the team, and I'm not sure your suggestion quite does this. We could say something like: 'This will trigger a high priority ticket which could alert the team during the night, so it should only be used for critical incidents.'

jonathanglassman commented on source/support.html.erb:
```diff
@@ -134,7 +138,24 @@ title: Support
-        details you have been sent.
+        details you have been sent.</p>
+
+        <h2 class="heading-large">Keep up to date</h2>
```
Perhaps change to "Keeping up to date" just to make consistent with the previous heading "Contacting us..."

jollery replied:
Yep, sounds good

jonathanglassman commented on source/support.html.erb:
```diff
@@ -134,7 +138,24 @@ title: Support
+      </p>
+
+      <p>
+        Join our <a href="https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en#!forum/gov-uk-paas-announce">announcements mailing list</a>
```
Potentially could say "Click here to join..." just to make it explicitly clear how they join the mailing list.

jollery replied:
Yes to the change

jollery committed:
Changes made after review

Jonathan Glassman has reviewed and suggested changes - this commit includes those changes.

paroxp commented on source/support.html.erb:
```diff
@@ -134,7 +138,24 @@ title: Support
+        <a href="https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en#!forum/gov-uk-paas-announce">Join our announcements mailing list</a>
+        to find out about new features and changes to the platform. If you have
+        trouble signing up, please contact
+        gov-uk-paas-support@digital.cabinet-office.gov.uk and we will add you
```
Recommend adding in email address link - apologies for missing this first time around.

paroxp reviewed (changes requested):
See one additional comment. Apart from that, all new content looks good on the web preview.

paroxp committed:
Fix the missing link

The email need to be clickable on the webpage.
